### ISSUE TITLE
Example not working

### ISSUE THREAD

**Issue by SteveDiamond** [cvxgrp/cvxpylayers#48]
I ran the example on the home page for PyTorch:
```
import cvxpy as cp
import torch 
from cvxpylayers.torch import CvxpyLayer

n, m = 2, 3
x = cp.Variable(n)
A = cp.Parameter((m, n))
b = cp.Parameter(m)
constraints = [x >= 0]
objective = cp.Minimize(0.5 * cp.pnorm(A @ x - b, p=1))
problem = cp.Problem(objective, constraints)
assert problem.is_dpp()

cvxpylayer = CvxpyLayer(problem, parameters=[A, b], variables=[x])
A_tch = torch.randn(m, n, requires_grad=True)
b_tch = torch.randn(m, requires_grad=True)

# solve the problem
solution, = cvxpylayer(A_tch, b_tch)

# compute the gradient of the sum of the solution with respect to A, b
solution.sum().backward()
```
I get gradient None. I am running the master branch of cvxpy and cvxpylayers. I assume this is a bug.

**Comment by akshayka** [comment]
@steveDiamond, I'm unable to reproduce your issue. Are you accidentally checking `A.gradient`, instead of `A_tch.grad`? PyTorch populates gradients in the `grad` attribute of Tensors.

**Comment by SteveDiamond** [comment]
Oops, that was it thanks!
